csharp/test/Microsoft.ML.OnnxRuntime.Tests/Microsoft.ML.OnnxRuntime.Tests.csproj

```diff
@@ -38,7 +38,7 @@
       <Visible>false</Visible>
     </None>
     
-    <None Include="$(buildDirectory)\$(Configuration)\models\**\*">
+    <None Include="$(buildDirectory)\models\**\*">
         <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
you could copy "PreserveNewest" instead of Always.

Reply:
Done